README.md

```diff
@@ -74,32 +74,26 @@ ckan-client-js
 Upload file from **NodeJS**
 
 ```js
-const { Client, Open } = require('./lib/index')
+const { Client } = require('./lib/index')
 
-const file = new Open.FileAPI.NodeFileSystemFile('file.csv')
 const client = new Client('key', 'organization-name', 'dataset-name', 'apiUrl')
 
-client
-  .ckanAuthz('http://localhost:5000')
-  .then(response => client.push(file, response.result.token))
+client._pushBlob(file)
```

Comment:
Why is this `_pushBlob` rather than `pushBlob` - it's not an internal method.

Reply:
I will fix it to use `pushBlob()`